src/walletconnect/walletconnect.test.js

```diff
@@ -9,6 +9,9 @@ import { getCryptoCurrencyById, setSupportedCurrencies } from "../currencies";
 import type { Account } from "../types/account";
 import { emptyHistoryCache } from "../account";
 import { setEnv } from "../env";
+import { setPlatformVersion } from "../platform/version";
+
+setPlatformVersion("0.0.1");
```

Comment:
walletconnect and platform are not related 🤔 

They could be factorized in the future 👍